sklearn/utils/metaestimators.py

```diff
@@ -10,11 +10,47 @@
 __all__ = ['if_delegate_has_method']
 
 
+def hasattr_nested(obj, attr):
+    """Check if obj has a (possibly nested) attribute
```

Comment:
`hasattr` doesn't work with nested attribute. `hasattr_nested` is like `attrgetter`, but only check if attribute is available and not accessing the attribute
In this case we are trying to check if the following statement is true:
```Python
hasattr_nested(GridSearchCV(), 'estimator.predict_proba')
```